Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt

```diff
@@ -91,7 +91,12 @@ class ScheduleAdapterItem internal constructor(
                     else View.INVISIBLE
             holder.sessionLayout.visibility = View.VISIBLE
             holder.title.text = itemData.talkTitle
-            holder.room.text = itemData.room
+
+            // WORKAROUND FOR https://github.com/Droidcon-Boston/conference-app-android/issues/165
+            // If the talk title is Check-In then hardcode the room name to be the lobby
+            if (itemData.talkTitle.toLowerCase().contains("check-in")) {
+                holder.room.text = "Calderwood Pavilion Lobby"
```

Comment:
Ugh...please don't do this.
Check that the other fields are null instead.  E.g., room.text is probably null in this case.  If that should not be null in Slidesup, fix it in slidesup.  I don't think it should be null.